Telegram/SourceFiles/platform/linux/specific_linux.cpp

```
@@ -245,6 +245,22 @@ void LaunchGApplication() {
 			.chopped(8)
 			.toStdString();
 
+		const auto owned = [&] {
+			if (!Gio::Application::id_is_valid(appId)) {
+				return false;
+			}
+
+			try {
+				return base::Platform::DBus::NameHasOwner(
+					Gio::DBus::Connection::get_sync(
+						Gio::DBus::BusType::SESSION),
+					appId);
+			} catch (...) {
+			}
```

The idea of ignoring all exceptions by ellipsis is not considered a good practice. It is better to catch only expected exceptions and log them to facilitate easier debugging.